privaterelay/settings.py

```diff
@@ -201,9 +201,16 @@
 DAYS_PER_BILLING_CYCLE = config("DAYS_PER_BILLING_CYCLE", 30, cast=int)
 MAX_DAYS_IN_MONTH = 31
 IQ_ENABLED = config("IQ_ENABLED", False, cast=bool)
+IQ_FOR_VERIFICATION = config("IQ_FOR_VERIFICATION", False, cast=bool)
 IQ_FOR_NEW_NUMBERS = config("IQ_FOR_NEW_NUMBERS", False, cast=bool)
+IQ_MAIN_NUMBER = config("IQ_MAIN_NUMBER", "")
 IQ_OUTBOUND_API_KEY = config("IQ_OUTBOUND_API_KEY", None)
 IQ_INBOUND_API_KEY = config("IQ_INBOUND_API_KEY", "")
+IQ_MESSAGE_API_ORIGIN = config(
+    "IQ_MESSAGE_API_ROOT", "https://messagebroker.inteliquent.com"
+)
```

Comment:
This is cleaner than the last PR. But,

1. When will the API Origin change? Is this premature flexibility?
2. Should the setting `IQ_MESSAGE_API_ORIGIN` match the env variable `IQ_MESSAGE_API_ROOT`

Reply:
1. The API origin could change if we use the iQ "sandbox" environment in local, dev, or stage.
2. I updated the env var to match setting name `IQ_MESSAGE_API_ORIGIN`